### DISCUSSION TITLE
"show displays as individual windows" not available on macOS

### DISCUSSION THREAD

**Discussion by stefaz** [rustdesk/rustdesk#8304] [General]
Hi all,

I'm using the latest 1.2.6 nightly build and trying to access a multi-monitor windows system from a macOS client.
I found that the option "Show displays as individual windows" is not available on macOS - is that supposed to only work between 2 windows systems? according to the PR https://github.com/rustdesk/rustdesk/pull/5945 it should also work from a macOS client, so I'm wondering what I'm missing here.
any ideas?

thanks!

**Comment by fufesou** [comment]
Please check if texture rendering is enabled? Multi ui sessions requires this feature.
macOS disable it by default.

![image](https://github.com/rustdesk/rustdesk/assets/13586388/46ccb0ea-43f0-4570-b918-97207bdea65b)

**Reply by stefaz** [reply]
great - that did the trick.
Thank you!

**Comment by fufesou** [comment]
I'll try implement multi ui sessions without texture rendering.
